webapp/src/components/backstage/playbook_runs/playbook_run/playbook_run.tsx

```diff
@@ -52,50 +52,52 @@ const useRHS = () => {
 };
 
 const PlaybookRunDetails = () => {
-    const {formatMessage} = useIntl();
     const dispatch = useDispatch();
     const match = useRouteMatch<{playbookRunId: string}>();
-    const currentRun = useRun(match.params.playbookRunId);
+    const playbookRunId = match.params.playbookRunId;
+
     const [playbookRun, setPlaybookRun] = useState<PlaybookRun | null>(null);
     const playbook = usePlaybook(playbookRun?.playbook_id);
     const [following, setFollowing] = useState<string[]>([]);
     const [fetchingState, setFetchingState] = useState(FetchingStateType.loading);
     const [playbookRunMetadata, setPlaybookRunMetadata] = useState<PlaybookRunMetadata | null>(null);
     const [statusUpdates, setStatusUpdates] = useState<StatusPostComplete[]>([]);
+    const myUser = useSelector(getCurrentUser);
 
     const RHS = useRHS();
+    const runFromStore = useSelector<GlobalState, PlaybookRun|undefined|null>(getRun(playbookRunId));
```

Comment:
Do we need all the explicit typing here?
```suggestion
    const runFromStore = useSelector(getRun(playbookRunId));
```